db/migrate/20170719204710_create_pieces.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.integer :game_id, index: true
```

Comment:
@mscoutermarsh I made the change. I was really nervous to change a migration file that I had already run locally (so I was trying to create yet another new migration file to change the pieces table and was coming up with local errors), but this seemed to work just fine. I'm going to go ahead and commit.